### PR TITLE
Restructure samples, corda and fabric-cli directories

### PR DESCRIPTION
**Pull request by sanvenDev**
1. Created two subdirectories: `samples/corda` and `samples/fabric`, and moved respective sample applications to correct folders.
2. Moved `fabric-cli` from deep inside `tests/network-setups` to `samples/fabric` folder.
3. Updated Getting Started Page, to point to new paths.
4. Replacing `<PATH-TO-NETWORK-SETUPS>` to `<PATH-TO-WEAVER>/tests`.

All 4 data-transfer flows and asset-exchange have been tested.

### CODE REVIEW

File: docs/docs/external/getting-started.md [hunk 1]
```diff
@@ -275,25 +275,25 @@ Build the interoperation Cordapp as follows:
...
 
 This is a simple Cordapp that maintains a state of type `SimpleState`, which is a set of key-value pairs (of strings).
-The code for this lies in the `samples/corda-simple-application` folder.
+The code for this lies in the `samples/corda/corda-simple-application` folder.
 
 #### Building
 
 Build the `corda-simple-application` Cordapp as follows:
-- Navigate to the `samples/corda-simple-application` folder.
+- Navigate to the `samples/corda/corda-simple-application` folder.
```
Comment (knarayan) [inline docs/docs/external/getting-started.md:283]: Can we rename `corda-simple-application` to `simple-application`?

**Review by knarayan — CHANGES_REQUESTED** [review]
Added one minor comment @sanvenDev .

File: docs/docs/external/getting-started.md (hunk 1 above)
Comment (knarayan) [reply docs/docs/external/getting-started.md:283]: Or to `simple-cordapp`?

**Review by sanvenDev — COMMENTED** [review]
Marking it resolved after Slack chat.

**Review by knarayan — APPROVED** [review]
Looks fine to me.

**Pull request merged**